lobbyist_employer.mk

```diff
@@ -0,0 +1,45 @@
+# Lobbyist expenditures and contributions
```

Comment:
This file is very, very similar to lobbyist.mk, with the primary differences being field names in the SQL / `csvjoin` statements.